src/settings.js

```diff
@@ -1392,6 +1392,10 @@ var AUTODEBUG = 0;
 // wasm normally, then compile that to JS).
 var WASM2JS = 0;
 
+// Whether we should link in the runtime for ubsan.
+// 0 means do not link ubsan, 1 means link minimal ubsan runtime.
+var UBSAN_RUNTIME = 0;
```

Comment:
It would be great if we didn't have to expose a new `-s` style flag for this feature, but I guess Settings is the idiomatic way to pass around config state. Can you add a comment here that the recommended way to use this feature is using the clang flags?